content/docs/cli-reference/help/logging.md

```diff
@@ -2,7 +2,6 @@
 title: logging
 name: logging
 url: /docs/cli-reference/help/logging/
-date: 2024-04-17T18:02:57+02:00
```

Comment:
Why do we delete the `date`? I admit that it is redundant when we use versioning with `git`, but I am wondering why now?